blueprints/extra-machines-resilient-apps.html.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Resilient apps use extra Machines
+layout: docs
+nav: firecracker
+---
+
+Fly Machines are fast-launching VMs; they're the compute of the Fly.io platform. Every Machine is assigned to a single physical host. This helps keep `start` operations consistently fast, but comes at the cost of Machine availability when something goes wrong on a single host.
+
+We have several platform features available to create high availability services with Machines. They all require a second Machine, even if it’s not running most of the time.
+
+Reasons to set up extra Machines:
+
+- Your app won't fail when a single host fails because another Machine will start up to take over app requests or tasks.
+- If your app has a lot of users, or bursts of high usage, then the Fly Proxy can load balance requests and automatically stop and start Machines based on traffic to your app. 
+- When Machines aren't running, you only pay for rootfs, not CPU and RAM. Machine rootfs is [cheap](/docs/about/pricing/#stopped-fly-machines), like 18 cents a month for an average Elixir app that uses 1.2 GB of rootfs.
+
+You can add [extra Machines](#extra-machines-for-apps-that-receive-requests) for Fly Proxy to start and stop as needed, which is great for apps that have built-in replication or that don't share data.
+
+For worker Machines that don't accept external requests, [create a standby Machine](#create-a-standby-machine), stopped and ready to take over in case the original Machine becomes unavailable.
```

Comment:
The thrust of this is that if the Machine can't be woken by the Fly Proxy you can use a standby Machine instead, right?

Reply:
Yes. I am removing this from the current location and into the respective section. I will reword it there.